src/algorithms/sorting/sort_struct_custom.md

```diff
@@ -0,0 +1,55 @@
+## Sort a Vector of Structs by custom criteria
+
+[![std-badge]][std] [![cat-science-badge]][cat-science]
+
+This example sorts a Vector of Person structs by age. This is achieved
+with [`vec:sort_by`] and providing it with custom comparator function.
+
+```rust
+use std::fmt;
+
+struct Person {
+	name: String,
+	age: u32
+}
+
+impl Person {
+    pub fn new(name: String, age: u32) -> Self {
+        Person {
+            name,
+            age
+        }
+    }
+}
+
+impl PartialEq for Person {
+    fn eq(&self, other: &Person) -> bool {
+        self.name == other.name
+    }
+}
+
+impl fmt::Debug for Person {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.name)
+    }
+}
+
+fn main() {
+    let expected = vec![
+        Person::new("Al".to_string(), 60),
```

Comment:
I'd propose to move the whole body of `expected` to the right side of `assert_eq` as in the previous examples

Reply:
I inlined the expected variables.